ros2bag/ros2bag/verb/record.py

```diff
@@ -208,6 +217,11 @@ def _check_necessary_argument(self, args):
 
     def main(self, *, args):  # noqa: D102
 
+        if args.topics_positional:
```

Comment:
Now it is possible to set args.topics_positional and args.topics at the same time.  
For this case, error should be reported.  

```python
if args.topics_positional and args.topics:
    return print_error('...')
```

Reply:
@Barry-Xu-2018 As regards
> Now it is possible to set args.topics_positional and args.topics at the same time.

That is actually is not possible because `args.topics_positional` and `args.topics` are in mutually exclusive group. See https://github.com/ros2/rosbag2/pull/1632/files#diff-69df89597d099d6eeeb32ecc480c22116dc6c97f32556ef9636eee901f62452cR62-R68 . The Python arguments parser will detect such situation if one will try to combine a positional argument with optional `--topics` and throw an error with the error message about it. I have checked it.

Reply:
Sorry. I didn't notice this line `topics_args_group = parser.add_mutually_exclusive_group()`.